src/controllers/message.js

```diff
@@ -17,4 +17,26 @@ const createMessage = async (req, res) => {
     res.status(StatusCodes.CREATED).json({ message })
 }
 
-module.exports = { createMessage }
+// Get message conversation
+const getMessageConversation = async (req, res) => {
+    const {
+        user: { userId },
+        params: { messagingPartnerUserId: receivedByUser },
+    } = req
+
+    const messages = await Message.find({
+        receivedByUser: [userId, receivedByUser],
```

Comment:
Won't this return any message where receivedByUser is userId or receivedByUser? This would include messages sent by anyone. 

I think what we want to do is find all messages where sentByUser = userId and receivedByUser = messagingPartnerUserId. Along with this we also want all messages where sentByUser = messagingPartnerUserId and receivedByUser = userId.

Reply:
Hi @jpremo, thank you for the feedback. I have pushed a new commit in response to your review. Could you please take a look at it? Thank you.